src/query/numeric_opfunc.h

```diff
@@ -40,7 +40,17 @@ typedef enum
   DATA_STATUS_NOT_CONSUMED = 1005	/* Operation not consumed all input */
 } DB_DATA_STATUS;
 
-#define NUMERIC_MAX_STRING_SIZE (80 + 1)
+/*
+ * NUMERIC_MAX_STRING_SIZE:
+ * The maximum input digits for numeric values is limited to 254 digits including
+ * sign, decimal point, etc. NUMERIC_MAX_STRING_SIZE is used for numeric parsing,
+ * arithmetic operations, scale adjustment, rounding, etc. It is set to twice
+ * the size of TWICE_NUM_MAX_PREC to accommodate digit increase due to scale adjustment.
+ * 
+ * Maximum string size for NUMERIC output: 175 * 2 = 350
+ * = (max fractional digits (38 + 127) + 10 extra digits) * 2
+ */
+#define NUMERIC_MAX_STRING_SIZE ((DB_MAX_NUMERIC_PRECISION + DB_MAX_NUMERIC_SCALE) + 10) * 2
```

Comment:
```suggestion
#define NUMERIC_MAX_STRING_SIZE  (((DB_MAX_NUMERIC_PRECISION + DB_MAX_NUMERIC_SCALE) + 10) * 2)
```
가능하면 습관적으로 괄호로 묶어 주세요.
나중에 이걸 사용할 때 의도치 않은 오류를 줄여 줍니다.

Reply:
네, 말씀하신 내용을 반영하여 수정했습니다.
추가로, TWICE_NUM_MAX_PREC 부분도 괄호를 사용하여 보완했습니다.